packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/WorklistDataGrid.1/About.2.doc.mdx

```diff
@@ -14,14 +12,102 @@ import WorklistDataGridColumnStates from './Examples.3/WorklistDataGridColumnSta
 Terra Data Grid is currently under alpha release. There may be breaking changes between releases until it is stable with the 1.0 release.
 </Notice>
 
+## Table of Contents
+* [Overview](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#overview)
+* [Design Decisions](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#design-decisions)
+* [Examples](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#examples)
+* [Accessibility](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#accessibility)
+* [Implementation](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#implementation)
+* [Terra Standards](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#terra-standards)
+* [Properties](/components/cerner-terra-framework-docs/data-grid/worklist-data-grid/about#properties)
+
+## Overview
+  The WorklistDataGrid is an [accessible grid](https://www.w3.org/WAI/ARIA/apg/patterns/grid/) component that allows the user to display data in a table like structure with rows and columns. The grid provides core functionality that a consumer would need while allowing flexibility for the consumers to control specific behavior of the grid based on their requirements.
```

Comment:
Since this is markdown, consecutive lines are still rendered as a paragraph. Can we have 1 sentence per line for better readability.
```suggestion
  The WorklistDataGrid is an [accessible grid](https://www.w3.org/WAI/ARIA/apg/patterns/grid/) component that allows the user to display data in a table like structure with rows and columns.
The grid provides core functionality that a consumer would need while allowing flexibility for the consumers to control specific behavior of the grid based on their requirements.
```

Reply:
Updated